infrastructure/transform/variables.tf

```diff
@@ -68,11 +60,26 @@ variable "deployer_ip_address" {
   type = string
 }
 
-variable "local_mode" {
+variable "tf_in_automation" {
   type = bool
 }
 
+variable "access_databricks_management_publicly" {
+  type        = bool
+  description = "Whether to allow access to the Databricks workspace management plane via a public network"
+  default     = true
+}
+
+variable "transform" {
+  description = "Transform configuration block (populated from root config file)"
+  type = object({
```

Comment:
if both vars are optional then should the whole block be optional too?

Reply:
Yep good point!